### ISSUE TITLE
Error starting Meta node after Genesis

### ISSUE THREAD

**Issue by Lenin1218** [ElrondNetwork/elrond-go#677]
I had a small error when starting a node on meta. The fact is that I did not have time to run this node before Genesis. I was 3-4 minutes late. No synchronization was performed. 

The reason at all not in feasts. Maybe there was a fork, as restarts did not help. Reinstalling and deleting the database did not help. I lost about 25 minutes not mining blocks. Maybe the catch will be awarded? ))

I did a few restarts later and with a few tries I managed to get it running. It's all right now. I don't know what it's about. The screenshots show the inscription in the logo.

I think developers need to look at the cause of the error and try to reproduce it.

"roll back behind final header is not permitted"
"account was not found"
"requested header with nonce 1 from network as probable highest is 208"
"starting roll back"
![000](https://user-images.githubusercontent.com/56969655/68952390-d5378900-07d0-11ea-8f02-a155d7cef33a.jpg)

![0000](https://user-images.githubusercontent.com/56969655/68952412-dc5e9700-07d0-11ea-909b-8306776af2df.jpg)

**Comment by joesixpack** [comment]
I just ran into this problem after scping the metanode to a new machine.  At first I thought it was because I started the node with the wrong keys but copying over the correct keys and wiping didn't help.

The highest nonce increasing too slow vs each round to ever catch up that I can see.

![image](https://user-images.githubusercontent.com/23145642/69107878-b7b02c80-0a27-11ea-9697-3de5cf3e67c6.png)
